modules/apps/content-dashboard/content-dashboard-test/src/testIntegration/java/com/liferay/content/dashboard/web/internal/portlet/test/ContentDashboardAdminPortletTest.java

```diff
@@ -1136,6 +1136,41 @@ public void testGetSearchContainerWithStatusScheduled() throws Exception {
 				LocaleUtil.US));
 	}
 
+	@Test
+	public void testGetSearchContainerWithTag() throws Exception {
```

Comment:
Should it be testGetSearchContainerWithAsset?

Reply:
These changes add the filter by tags to the Content Dashboard. The filter dropdown name is `Tags...` so I am ok with the actual name of the test. What do you think @cgoncas?

Reply:
As @dgarciasarai says, the name of the test is related to the Request param, no with the actual implementation of it.

That is the reason we have WithStatus, and no WithVersion, or WithScope no WithGroupId.

But I think we can wait until https://github.com/liferay-tango/liferay-portal/pull/54 is merged, that follows the same pattern, and see whats Brian prefer for the name.

Reply:
See https://github.com/brianchandotcom/liferay-portal/pull/91720#issuecomment-663334709

I think it the test should be called `*WithAssetTag()` since Brian mentioned it in the filter by categories.

Reply:
DONE :)